Bugsnag.podspec.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Bugsnag",
-  "version": "5.23.2",
+  "version": "5.22.3",
```

Comment:
Shouldn't these be `5.23.3`?
```suggestion
  "version": "5.23.3",
```

Reply:
I've provided Github suggestions for these, but actually it would be better to change them all in a single commit.

Reply:
Good catch - I've updated these in a single commit.